src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-name/table-interactor-name.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="container">
+  <div class="nameAndIcons" [ngClass]="{inlineDisplay: interactorType==='stable complex'}">
+    <i *ngIf="organismIconDisplay"
+       class="{{interactorOrganismIcon}}"
+       title="{{interactorOrganism}}"></i>
+
+    <i *ngIf="interactorTypeDisplay"
+       class="{{interactorTypeIcon}}"
+       title="{{interactorType}}"></i>
+
+    <div class="name" title="{{interactorName}}" *ngIf="IDDisplay"> {{ interactorName }}
+    </div>
+    <div class="name" title="More information about {{interactorId}}" *ngIf="!IDDisplay">
+      <a *ngIf="!IDDisplay" href="{{identifierLink}}" target="_blank"
+         class="externalLinkContainer">
+        {{ interactorName }}
+      </a>
+    </div>
+
+    <a [routerLink]="['/complex/search']"
+       [queryParams]="{query: interactorId, page: 1}"
+       [preserveFragment]="true"
```

Comment:
Because we now check the number of results to decide between list or navigator view, it makes sense to get rid of this line. So, instead preserving the fragment, when people click on search of one of the interactors, it opens a new tab, and that tab is list or navigator view depending on the number of results.